packages/paste-core/components/anchor/src/types.ts

```diff
@@ -1,11 +1,15 @@
-import * as PropTypes from 'prop-types';
-import {LayoutProps, SpaceProps} from '@twilio-paste/style-props';
+import type {LayoutProps, SpaceProps} from '@twilio-paste/style-props';
+import type {BoxProps} from '@twilio-paste/box';
 
 export type AnchorTabIndexes = 0 | -1;
 export type AnchorTargets = '_self' | '_blank' | '_parent' | '_top';
 export type AnchorVariants = 'default' | 'inverse';
 
-export interface AnchorProps extends React.AnchorHTMLAttributes<HTMLAnchorElement>, LayoutProps, SpaceProps {
+export interface AnchorProps
+  extends React.AnchorHTMLAttributes<HTMLAnchorElement>,
+    LayoutProps,
+    SpaceProps,
+    Pick<BoxProps, 'element'> {
```

Comment:
Why did you add `element` from `BoxProps` instead of adding it manually below? Is it so that the comment would carry over without having to duplicate it? ( /** element identifier for customization */)

Reply:
It's generally a good practice to have a "single source of truth". This way, if we ever want to change the type of `element`, we won't have to change every instance of it in every package. 

That said, I'm not sold that Box is the best source of truth, maybe it should be our from our types package instead.

Reply:
Makes sense, I wonder if we should all align on where we're pulling `element` from since we're doing so many of these changes. I added it directly to `FlexProps` but I can do a refactor before merging.

Reply:
I don't think we should get too hung up about it for now. It's unlikely to ever change and shouldn't have a material impact on customers. Feel free to redeclare it as you like.

Reply:
The way I put is that we're extending or exposing Box functionality via this components API, so it makes sense to me to extend from `BoxProps`. But its a string so it's no big deal